middlewares/conductor/station_events.rb

```diff
@@ -5,31 +5,31 @@ class Conductor
   # Provides `station` as a hash of new attributes for a Station object. The
   # attributes given in this hash will always be enough to create a new Station
   # from scratch.
-  register_handler 'stations', :update do |channel, args, kwargs|
+  register_handler 'stations', :update do |event|
     # No conducting is necessary
   end
 
   # activate -> [station] {**defaults}
   #   once
   # Sent when a Station becomes publicly visible. `station` will be an
   # attributes hash equivalent to that in the update event.
-  register_handler 'stations', :activate do |channel, args, kwargs|
+  register_handler 'stations', :activate do |event|
     # No conducting is necessary
   end
 
   # deactivate -> [station] {**defaults}
   #   once
   # Sent when a Station stops being publicly visible. `station` will be an
   # attributes hash equivalent to that in the update event.
-  register_handler 'stations', :deactivate do |channel, args, kwargs|
+  register_handler 'stations', :deactivate do |event|
     # No conducting is necessary
   end
 
   # depart -> [vehicle] {**defaults}
   #   once
   # Sent when a vehicle departs from this Station. `vehicle` will be the
   # attributes of the vehicle that departed.
-  register_handler 'stations', :depart do |channel, args, kwargs|
+  register_handler 'stations', :depart do |event|
```

Comment:
I added those just for consistency and clarity.

With these, Conductor has a handler implementation (granted, some empty ones) for every event on the "WAMP Events and RPCs" wiki page.

They could easily be left for `default_event_handler` to take care of, but adding all of them for completeness wasn't much extra trouble.